ui/src/single-spa-entry.js

```diff
@@ -27,10 +27,18 @@ const reactLifecycles = singleSpaReact({
 
 export const { bootstrap } = reactLifecycles;
 export const mount = (props) => {
-  // Get the full path, querystring and hash. The regex gets the first
-  // forward slash that is not a double forward slash and everything after it.
-  const matches = window.location.href.match(/(?<!\/)\/(?!\/).+/);
-  // The app should never reach this entrypoint witout the basename and react
+  // Get the full path, querystring and hash (location.pathname can't be used as
+  // it does not contain the querystring and hash).
+  const matches = window.location.href
```

Comment:
You could make this a bit easier to understand for those that don't know regex by using a string replace to remove the protocol and hostname. 

```js
const loc = window.location;
const matches = loc.href.replace(`${loc.protocol}//${loc.host}`, '');
```
or you can reconstruct it
```js
const loc = window.location;
const matches = loc.pathname + loc.search + loc.hash;
```

Reply:
I've done this and it's a lot cleaner, would you mind doing another review/QA?